src/main/java/org/opentripplanner/transit/model/timetable/Trip.java

```diff
@@ -20,6 +20,18 @@
 import org.opentripplanner.transit.model.network.Route;
 import org.opentripplanner.transit.model.organization.Operator;
 
+/**
+ * A Trip represents the movement of a public transport vehicle on a given {@link Route}, using a
+ * given {@link TransitMode}, on a given sequence of stops served at given passing times.
+ * <p>A Trip can run at most once per service date.
```

Comment:
Frequency based trips?

Reply:
Should be reworded as :

> A scheduled Trip can run at most once per service date, while a frequency-based Trip runs several times on a given service date.

Reply:
This sounds good. If you add this, I will approve.

Reply:
Done